### PR TITLE
Ethari Rework

### PR DESCRIPTION
**Pull request by Ziiro**
:cl:
balance: Digitgrades can now wear clown shoes. AND ONLY CLOWN SHOES.
balance: Ethari are now Digitigrade.
balance: Ethari CAN NO LONGER USE GUNS
balance: Ethari are no longer slowed down by things like backpacks, hardsuits, or the like
balance: Ethari now have limited darkvision, and take double damage/stun from flashes
balance: Ethari now roll their R's.
balance: Ethari now take 50% less cold (Instead of ~33% less)
balance: Ethari now take 150% more heat damage (up from 130%)
balance: Ethari now take normal brute damage (used to be 20% MORE)
balance: Ethari now take 150% more burn damage (up from 120%)
/:cl:

[why]: Re-configuring them to an "Uplifted Beast race." meant some changes.

Things I'm on the fence about:
* Thinking about blocking gloves, due to their messed up hands. (Presumed to be the larger fingers/claws). However this makes insulating them 100% out of reach. Which is a huge deal.
* The ignoring slowdown: If I'm going to force them to be melee only, they have to have *something* to prevent from being kited too badly if they wear hardsuits/etc. It's also a nice perk, but.. dangerous to use. Speeding them up wholesale is out of the question.
* The general damage numbers: I'd like to make them more "tanky" because the loss of guns is a pretty big deal. I'm hesitant to make them more fire weak, because I already did that schtick for IPCs.

### CODE REVIEW

**Comment by SmokingKilz** [comment]
Only two concerns, this makes sec Ethari pretty much useless to any form of lethal threat & antag Ethari should be able to use weapons. Apart from that, I like the changes so far.

**Comment by Iamgoofball** [comment]
I'm not seeing a removal here anywhere? Doesn't seem like a good rework to me.

**Comment by scrubmcnoob** [comment]
pr has been blessed by goof

**Comment by Ziiro** [comment]
>Only two concerns, this makes sec Ethari pretty much useless to any form of lethal threat

Armor up and grab a baton if it cat be stunned. If it can't, better make a chainsaw. You can outrun most targets even in heavy armor, and injury does not slow them down.

>antag Ethari should be able to use guns. Apart from that, I like the changes so far.

Strong disagree. Smart antagonists will plan around it. It also would remove a bit of flavor if they could just go around a pretty defining trait like that.

>I'm not seeing a removal here anywhere? Doesn't seem like a good rework to me.

Hey buddy, I think you've got the wrong PR. Removal PR is two blocks down.

**Comment by FlattestGuitar** [comment]
I'm fine with the majority of these changes, but:
* If Ethari cannot use guns they are suddenly unable to play as remotely efficient Miners or Security
* Giving them +2.5 attack damage seems like a meaningless change. It won't come into use outside of greytide fistfights, which we want to discourage to start with.
* There are some hardcore consequences of making a species immune to ALL slowdown effects. Low health, hunger, pieces of equipment that use massive slowdown for balance are all effectively cut out. This leaves far too much room for powergamey shit.

Also, I heavily disagree with changing any of this functionality just for antagonists. That will enable security to just hand them a practice laser and prove they're not an antag, we don't want that. Clowns get an RNG roll, so it's far less reliable, but here the story is different.

**Comment by Xantholne** [comment]
Can't say I agree with them not being able to use guns, an alternative can probably be made but without firearms they're pretty shite. An exception I could see is they spawn with a trigger guard mod or a modified taser/KA with a trigger guard already allowing them to fire it. But I am not entirely sure if that is even possible for a taser or anything like that. Likely something we'd have to code in later.

**Comment by SmokingKilz** [comment]
I still don't see how arming up against a lethal threat is going to make you useful as a melee Ethari, especially in a game that revolves heavily around stuns. 

"Wizard? Welp, I'm fucking useless, maybe I'll try and clam his eyes ou- Aaand I'm dead."
"Giant spiders? Welp I'm either going to take a shit tonne of damage or potentially die because this spider hits like a truck."
"Nukies? Well I might as well fucking not even try at this point."

**Comment by Birdtalon** [comment]
Not being able to use guns puts the race at a massive disadvantage verses literally anything else. That is not a good change and is game breaking for ethari players.

**Comment by Ziiro** [comment]
I view the tradeoff of NOGUNS + NOSLOWDOWN to be an interesting one, but I get where you're all coming from on this.

However, without a major **something** for Ethari, this PR is mostly just pointlessly moving numbers around and making them digitigrade. I'm not sure that's enough to warrant keeping them.

**Comment by SmokingKilz** [comment]
If we figure out a way to keep them, I do enjoy the idea of the rolling their R's.

**Comment by Iamgoofball** [comment]
Do you people not understand no slowdown  is insanely good for melee w/riot armor

**Comment by imsxz** [comment]
christ you're all acting like combat, especially GUN combat, is the main focus of ss13... no slowdown is such a stupidly strong trait and you'd be blind to not see this. i'd happily trade away the guns that i only use 1 out of 10 rounds for the ability to not be slowed by swearing duffelbags, a hardsuit, damage, stamina, getting a leg cut off, and more all at the same time....

tl;dr this is a silly ethari buff and ziiro is in on the furry conspiracy

**Comment by Ziiro** [comment]
new ploy: increase powergamers self-loathing by forcing them to play a furry race

**Comment by swagbringer** [comment]
Are you guys going to testmerge this? I think trying it out for a week would be nice. If they turn out to be horrendous killing machines then jut revert it, and if they are weak, well no one is forcing anyone to play them.

**Comment by Ziiro** [comment]
Ruled over the top by maintainers, closing

Proceed with removal
